src/igvfd/audit/file_set.py

```diff
@@ -123,7 +154,7 @@ def audit_inconsistent_seqspec(value, system):
     '''
     [
         {
-            "audit_description": "Sequence files in a file set from the same sequencing run, flowcell_id, lane, and index are expected to link to the same seqspec file, which should be unique to that set of sequence files.",
+            "audit_description": "Sequence files in a file set from the same sequencing run (with the exception of non-single cell data), flowcell id, lane, and index, the combination of which define a sequencing set, are expected to link to the same seqspec file, which should be unique to that set of sequence files.",
```

Comment:
I am open to suggestions on how to alternatively word this mouthful.

Reply:
I don't know if it's any clearer by moving the list of properties into its own sentence? The description becomes much longer here, though.
> Sequence files in a file set from the same  sequencing set, are expected to link to the same seqspec file, which should be unique to that set of sequence files. A sequencing set is defined by the combination of flowcell id, lane, and index for non-single cell data, or flowcell ID, lane, index, and sequencing run for single cell data.

Reply:
thanks